components/WarningNote/WarningNoteRecap/WarningNoteRecap.tsx

```diff
@@ -5,13 +5,47 @@ import { useWarningNote } from 'utils/api/warningNotes';
 import { formStepsAdult, formStepsChild } from 'data/forms/warning-note';
 import { Resident } from 'types';
 import reviewFormSteps from 'data/forms/warning-note-review';
+import { FormComponentStep, FormStep } from 'components/Form/types';
 
 export interface Props {
   person: Resident;
   warningNoteId: number;
   formName?: string;
 }
 
+export const modifyReviewSummary = (): FormStep[] => {
+  const reviewedBy: FormComponentStep = {
+    component: 'TextInput',
+    name: 'lastModifiedBy',
+    label: 'Review done by',
+  };
+
+  const endDate: FormComponentStep = {
+    component: 'DateInput',
+    name: 'endDate',
+    label: 'End Date',
+  };
+
+  const nextReviewDateWithoutReviewDecision: FormComponentStep = {
+    component: 'DateInput',
+    name: 'nextReviewDate',
+    label: 'Next Review Date',
+  };
+
+  const updatedReviewFormSteps = reviewFormSteps;
+
+  updatedReviewFormSteps[0].components.pop();
```

Comment:
This `pop` and `push` feels a little unexpected (and I can see a few potential areas where this might fail), could you talk me through what you're trying to achieve here?